test/reline/test_key_actor_emacs.rb

```diff
@@ -138,11 +138,24 @@ def test_em_delete_prev_char_for_mbchar_by_plural_code_points
     assert_line_around_cursor("か\u3099", '')
   end
 
+  def test_bracketed_paste_insert
+    input_keys("d\C-aa")
+    input_key_by_symbol(:insert_multiline_text, char: "abc\n\C-abc")
+    assert_line_around_cursor("\C-abc", 'd')
+  end
```

Comment:
Q: Why does this code serve as a test for bracketed_paste_insert?

Reply:
`test_key_actor_emacs.rg` is testing the behavior of LineEditor with key input mainly registered by `key_actor/emacs.rb`.
Bracketed paste (start_escape_sequence, content, end_escape_sequence) is converted into `Key.new(content, :insert_multiline_text)` in reline and passed to LineEditor.
As a test for LineEditor, we need to input this key to LineEditor.

Test of handling `"\e[200~content\e[201~"` is done in `test_rendering.rb`

I also added small tweak to the test:
```ruby
input_keys("d\C-aa")
# ↓
set_line_around_cursor('a', 'd') # (and change it to 'A', 'Z')
```